### ISSUE TITLE
28577248: Identifier of Apple Watch should be selectable or copyable via cmd-c in the Devices window. Had to use Accessibility Inspector to copy it. O.o

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#16014]
#### Description

Peter Steinberger01-Oct-2016 07:37 AM

Summary:
Currently the best way to get the Identifier of the Apple Watch via Xcode is the Accessibility Inspector.

Steps to Reproduce:
Buy pretty Apple Watch
Try to run new app
See that provisioning profile complains about watch not in there
To to figure out where UUID is so you can add it to the portal
Hear the "bong" tone when you press cmd-c after selecting the watch.
Manually type it off? Nope! Accessibility Inspector it is.
Figure out that you're not alone and that many people do it that way
(https://twitter.com/steipete/status/782089464428556288)

Expected Results:
Should be select or copy/able. One can select the cell but not copy the content: https://twitter.com/steipete/status/782088687718916096

Actual Results:
Nopes.

Version:
Xcode 8.0 8A218a

Notes:
See my tweet storm for screenshots and rants :) 

https://twitter.com/steipete/status/782086870809403392

Configuration:

Attachments:
## 

Product Version: Xcode 8.0 8A218a
Created: 2016-10-01T05:38:05.517190
Originated: 2016-10-01T07:37:00
Open Radar Link: http://www.openradar.me/28577248
